### PR TITLE
[315] Add Cypress test infrastructure

### PR DESCRIPTION
**Pull request by gdaniel**
Bug: https://github.com/eclipse-syson/syson/issues/315

### CODE REVIEW

File: integration-tests/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AxelRICHARD) [inline integration-tests/.vscode/settings.json:1]: Is this modifications related to Cypress?

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@
 - Switch to Sirius Web 2024.5.0
 - Switch to @ObeoNetwork/gantt-task-react 0.4.9
 - Add dependency to org.apache.commons.commons-text 1.10.0
+- Add dependency to Cypress 12.11.0
```
Comment (AxelRICHARD) [inline CHANGELOG.adoc:14]: Could you also update the github actions please?
The goal is to execute the cypress tests for every PR, as we do in sirius web.

Comment (AxelRICHARD) [inline CHANGELOG.adoc:14]: You will also have to move this in the new v2024.7.0 section in the changelog after the merge of https://github.com/eclipse-syson/syson/pull/318, which should occurs tomorrow morning first task

File: integration-tests/.vscode/settings.json (hunk 1 above)
Comment (gdaniel) [reply integration-tests/.vscode/settings.json:1]: We have such file in the root of our repository. 
In Sirius Web it is also present in the `integration-tests` directory, I assume this is here to make sure settings are correct even if `integration-tests` is opened as a standalone project. Let me know if I should remove it

Comment (AxelRICHARD) [reply integration-tests/.vscode/settings.json:1]: Ok, I see. Let's keep it then.

File: CHANGELOG.adoc (hunk 2 above)
Comment (gdaniel) [reply CHANGELOG.adoc]: done

Comment (gdaniel) [reply CHANGELOG.adoc]: done
